3_javascript/javascript1/week2/homework2ndweek/calendar.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Calendar</title>
+  </head>
+  <body>
+    <script>
+      const todayDate = new Date();
+      const todaysWeekDay = todayDate.getDay();
+      const weekDays = [
+        "Sunday",
+        "Monday",
+        "Tuesday",
+        "Wednesday",
+        "Thursday",
+        "Friday",
+        "Saturday",
+      ];
+
+      function getEventDay(day) {
+        const someDay = todaysWeekDay + day;
+        if (someDay > 7) {
+          const thatDay = someDay % 7;
+
+          console.log(weekDays[thatDay]);
```

Comment:
return intead of console log.

Reply:
I did it with another mentor and thats what i was told to use, is there any specific reason why not to use console log? it doesnt work?

Reply:
it works, and for now you are all fine.
I am sorry to make this a bit complicated, but in real-life coding, functions usually return values, they never console.log them.
What you did is 100% correct for now.